packages/easy-coding-standard/build/target-repository/tests/Fixtures/pow.txt

```diff
@@ -0,0 +1,5 @@
+<?php
+echo pow(2, 3);
+-----
+<?php
+echo 2** 3;
```

Comment:
I don't think `.txt` files are recognized as PHP

Reply:
The approach is coppied from some of your old test. I guess I'll have to check how you do it these days then...